templates/manage_spec.epp

```diff
@@ -0,0 +1 @@
+I am a template
```

Comment:
I'm not a big fan of having a template, just for the tests. But I also don't have a better idea right now :thinking:

Reply:
Replaced with a function mock/stub